src/main/java/cdw/springtraining/gatekeeper/entites/Resident.java

```diff
@@ -0,0 +1,58 @@
+package cdw.springtraining.gatekeeper.entites;
+
+import jakarta.persistence.*;
```

Comment:
import only needed package and avoid importing all packages

Reply:
removed wildcard imports